backend/push/src/config/amqp-connection.ts

```diff
@@ -0,0 +1,15 @@
+import { CONFIG } from './config';
+import { connect, Channel } from 'amqplib';
+import { AmqpQueue } from '~/shared/enums/enums';
+
+const amqpConnect = async (): Promise<Channel> => {
+  const amqpServer = CONFIG.rabbitmqUrl;
+
+  const amqpChannel = await connect(amqpServer).then((connection) => connection.createChannel());
```

Comment:
I didn't see the logic to reconnect if the rabbitmq not started yet

Reply:
I resolved it in docker-compose with 
restart: on-failure:5

Reply:
I don't recommend doing it this way.